src/stan/services/variational/print_progress.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef STAN_SERVICES_VARIATIONAL_PRINT_PROGRESS_HPP
+#define STAN_SERVICES_VARIATIONAL_PRINT_PROGRESS_HPP
+
+#include <stan/services/io/do_print.hpp>
+#include <cmath>
+#include <iomanip>
+#include <iostream>
+#include <string>
+
+namespace stan {
+  namespace services {
+    namespace variational {
+
+      void print_progress(const int m,
```

Comment:
I'm channeling @bob-carpenter. The arguments for `int`, `bool`, and other primitive types don't need `const`. Just replace `const int m` with `int m`.